lib/test_helper/rollbax_fake.ex

```diff
@@ -0,0 +1,27 @@
+defmodule ElixirTools.RollbaxFake do
+  use ElixirTools.ContractImpl, module: Rollbax
+
+  @impl true
+  def report_message(level, message, custom \\ %{}, occurrence_data \\ %{}) do
+    ensure_no_struct(custom)
```

Comment:
I think we have to ensure that `occurrence_data` and `custom` are not structs but maps on the top level (because now we can pass number and it will work, not sure about dialyzer though)

Reply:
it's true, but I don't want to implement all checks of Rollbar, just the ones that are not obvious and causing therefore issues in production. Wdyt?